launch_test.go

```diff
@@ -745,6 +745,7 @@ func TestSetEnv(t *testing.T) {
 		"SD_BUILD_URL":           "https://api.screwdriver.cd/v4/builds/1234",
 		"SD_STORE_URL":           "https://store.screwdriver.cd/v1/",
 		"SD_UI_URL":              "https://screwdriver.cd/",
+		"SD_UI_BUILD_URL":        "https://screwdriver.cd/builds/1234",
```

Comment:
If the intention is to use this end point to further construct a link to Artifacts url, tgeb this won't give you want you want if. You need the full url eg: `https://screwdriver.cd/pipelines/300461/builds/6018884` You do have the pipeline id, so use that to construct full URL.

Reply:
there is some magic happening where opening url
https://screwdriver.cd/builds/6018884 would take me directly to the page even without the pipelineId. 

Is /pipelines/{pipelineId} the standard url, i didn't find any `pipelines` string in launch.go 🤔

Reply:
it does, but it doesn't support more path params. so with this `$SD_UI_BUILD_URL/artifacts/result.html` won't work.

Reply:
you have https://github.com/screwdriver-cd/launcher/pull/328/files#diff-338987301073a98ce4db3faf2cc35204R419, so `pipeline id` is available

Reply:
`/pipelines/{pipelineId}` is something that might be related to the UI component, is it ok to generate this url here?

Reply:
@jithine made the changes per your feedback to include `/pipelines/{pipelineId}`